docs/data-sources/clingen.mdx

```diff
@@ -0,0 +1,174 @@
+
+:::info Publication
+Savatt, J. M., Azzariti, D.R., Faucett, W.A., Harrison, S., Hart, J., Kattman, B., Landrum, M.J., Ledbeter, D.H., Rangel Miller, V., Palen, E., Rehm, H.L., Rhode, J., Turner, S., Vidal, J.A., Wain, K.E., Riggs, E.R., and Martin, C.L. (2018).  **ClinGens GenomeConnect Registry Enables Patient-centered Data Sharing.** *Human Mutation, 39(11),1668-676. doi:10.1002/humu.23633*.
+:::
+
+## ISCA Regions
+
+### TSV Extraction
+ClinGen contains only copy number variation variants, since the coordinates in ClinGen original file follow the same rule as BED format, the coordinates had to be adjusted to [BEGIN+1, END].
+
+```scss
+#bin    chrom   chromStart      chromEnd        name    score   strand  thickStart      thickEnd        attrCount       attrTags        attrVals
+nsv530705       1       564405  8597804 0       1       copy_number_loss        pathogenic      False   Developmental delay AND/OR other significant developmental or morphological phenotypes
+nsv530706       1       564424  3262790 0       1       copy_number_loss        pathogenic      False   Abnormal facial shape,Abnormality of cardiac morphology,Global developmental delay,Muscular hypotonia   HP:0001252,HP:0001263,HP:0001627,HP:0001999,MedGen:CN001147,MedGen:CN001157,MedGen:CN001482,MedGen:CN001810
+nsv530707       1       564424  7068738 0       1       copy_number_loss        pathogenic      False   Abnormality of cardiac morphology,Cleft upper lip,Failure to thrive,Global developmental delay,Intrauterine growth retardation,Microcephaly,Short stature       HP:0000204,HP:0000252,HP:0001263,HP:0001508,HP:0001511,HP:0001627,HP:0004322,MedGen:C0349588,MedGen:C1845868,MedGen:C1853481,MedGen:C2364119,MedGen:CN000197,MedGen:CN001157,MedGen:CN001482
+nsv533512       1       564435  649748  0       1       copy_number_loss        benign  False   Developmental delay AND/OR other significant developmental or morphological phenotypes
+nsv931338       1       714078  4958499 0       1       copy_number_loss        pathogenic      False   Developmental delay AND/OR other significant developmental or morphological phenotypes
+nsv530300       1       728138  5066371 1       0       copy_number_gain        pathogenic      False   Abnormality of cardiac morphology,Cleft palate,Global developmental delay       HP:0000175,HP:0001263,HP:0001627,MedGen:C2240378,MedGen:CN001157,MedGen:CN001482
+```
+#### Status levels
+* benign
+* curated benign
+* curated pathogenic
+* likely benign
+* likely pathogenic
+* path gain
+* path loss
+* pathogenic
+* uncertain
+
+#### Parsing
+We parse the ClinGen tsv file and extract the following:
+* chrom
+* chromStart (note this a 0-based coordinate)
+* chromEnd
+* attrTags
+* attrVals
+
+*attrTags* and *attrVals* are comma separated lists. *attrTags* contains the field keys and *attrVals* contains the field values. We will parse the following keys from the two fields:
```

Comment:
When mentioning fields, use the inline code notation (i.e. attrTags surrounded by backticks)